vmdk_plugin/main.go

```diff
@@ -151,7 +152,7 @@ func main() {
 		}
 		driver = photon.NewVolumeDriver(*targetURL, *projectID,
 			*vmID, mountRoot)
-	} else if *driverName == vmdkDriver {
+	} else if *driverName == vmdkDriver || *driverName == vsphereDriver {
```

Comment:
Can we print a message to warn user that "vmdk" driver is deprecated (print a warning message, but still continue to run the command)?